### PR TITLE
Use curl from runtime

### PR DESCRIPTION
**Pull request by Erick555**
There were couple security vulns: https://curl.haxx.se/docs/vuln-7.59.0.html
Other changes: https://curl.haxx.se/changes.html#7_61_0

### CODE REVIEW

**Comment by TingPing** [comment]
bot, build

**Comment by TingPing** [comment]
It is possible we no longer need to bundle this if the new runtime curl is acceptable. Somebody should investigate that.

**Comment by Erick555** [comment]
Those [symlinks](https://github.com/flathub/com.spotify.Client/blob/master/com.spotify.Client.json#L107) are still needed. If it's possible to generate them from runtime curl then we can get rid of curl from this package.

**Comment by TingPing** [comment]
Yea you can symlink the /usr lib into /app

**Comment by Erick555** [comment]
I'll check what can be done.

**Comment by Erick555** [comment]
@TingPing Succes, curl from runtime works well for us! The only problem is that path to lib seems [architecture dependend](https://github.com/flathub/com.spotify.Client/pull/54/files#diff-83fd27ba24db1f36e7de08ce56331feaR110). Do you have idea how to make it work for i386 as well?

File: com.spotify.Client.json [hunk 1]
```diff
@@ -136,7 +106,8 @@
                 "install spotify-bin /app/bin/spotify",
                 "install -Dm644 com.spotify.Client.appdata.xml /app/share/appdata/com.spotify.Client.appdata.xml",
                 "cp /usr/bin/ar /app/bin",
-                "ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib"
+                "ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib",
+                "ln -s /usr/lib/x86_64-linux-gnu/libcurl.so.4.5.0 /app/lib/libcurl-gnutls.so.4"
```
Comment (TingPing) [inline com.spotify.Client.json]: We do ship a 32bit build, not sure if it even needs the same version of curl since its old though.

Comment (TingPing) [reply com.spotify.Client.json]: Oh you already commented on that. The `ARCH_TRIPLE=$(gcc --print-multiarch)` usage above should do.

**Comment by Erick555** [comment]
It's ready! I Tested on x86_64. You may squash it.

**Pull request merged**